src/main/java/org/folio/bulkops/batch/jobs/processidentifiers/BulkEditInstanceIdentifiersJobConfig.java

```diff
@@ -111,6 +113,8 @@ public Step bulkEditInstanceStep(FlatFileItemReader<ItemIdentifier> csvItemIdent
       .processor(bulkEditInstanceProcessor)
       .faultTolerant()
       .skipLimit(1_000_000)
+      .retry(ConverterException.class)
+      .retryLimit(1_000_000)
```

Comment:
1_000_000 - please, don't forget to analyze and change this number to more reasonable.